auth/src/main/res/values/colors.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="colorPrimary">#3F51B5</color>
-    <color name="colorPrimaryDark">#303F9F</color>
-    <color name="colorAccent">#FF4081</color>
+    <color name="fui_colorPrimary">#3F51B5</color>
```

Comment:
It turns out I was wrong.  We can rename the `color` items just not the attribute of the theme in the style XML file. They just reference these colors by name and there's nothing special about the names.